obelisk_ws/src/cpp/obelisk_leap_control_cpp/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I know we've been pretty inconsistent about this throughout the repo, but I'm in favor for now of removing these license declarations in individual packages.

Also, right now, I chose the MIT license, so this license isn't compatible with the outermost declaration.